WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -1056,20 +1061,40 @@ local methods = {
       self:Enable();
     end
   end,
-  ["SetDragging"] = function(self, data, drop)
+  ["SetDragging"] = function(self, data, drop, displayButtons)
```

Comment:
This is the wrong place to implement most of this code.

This is the "SetDraggin" method of a single button. And that one should not care about other buttons.

You should move most of this code to WeakAuras.SetDragging